src/commands.cc

```diff
@@ -1485,6 +1485,35 @@ const CommandDesc debug_cmd = {
             write_to_debug_buffer(format(" * {}:\n{}",
                                   parser[1], dump_regex(compile_regex(parser[1], RegexCompileFlags::Optimize))));
         }
+        else if (parser[0] == "registers")
+        {
+            write_to_debug_buffer("Register info:");
+            const auto& register_manager = RegisterManager::instance();
+            for (Codepoint c : StringView{"\"/@^|%.#_:\\0123456789abcdefghijklmnopqrstuvwxyz"})
```

Comment:
Add a `begin/end` const method to RegisterManager that return `m_registers.begin/end`, then this line can become `for (auto&& [name, reg] : RegisterManager::instance())`.